src/Query.php

```diff
@@ -72,10 +77,17 @@ function url_parameters() {
      */
     function request_parameters() {
         if(empty($this->search_query)) {
-            throw new \Twingly\QueryException("Missing pattern");
+            throw new QueryException("Missing pattern");
         }
+
+        if(!empty($this->pattern)) { // handle deprecated variable
+            trigger_error('Query#pattern is deprecated, use search_query instead', E_USER_DEPRECATED);
+            $this->search_query = $this->pattern;
+        };
```

Comment:
This doesn't really work. When only setting `#pattern` I get the following error:

```
PHP Fatal error:  Uncaught Twingly\QueryException: Missing pattern in twingly-search-api-php/src/Query.php:80
Stack trace:
#0 /Users/mattias/repos/twingly-search-api-php/src/Query.php(71): Twingly\Query->request_parameters()
#1 /Users/mattias/repos/twingly-search-api-php/src/Query.php(51): Twingly\Query->url_parameters()
#2 /Users/mattias/repos/twingly-search-api-php/src/Client.php(95): Twingly\Query->url()
#3 /Users/mattias/repos/twingly-search-api-php/src/Client.php(78): Twingly\Client->get_response(Object(Twingly\Query))
#4 /Users/mattias/repos/twingly-search-api-php/src/Query.php(63): Twingly\Client->execute_query(Object(Twingly\Query))
```

I was expecting everything to work, but with `Query#pattern is deprecated, use search_query instead` shown in the output. I think this is just a matter of switching place of the two if-statements.